openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -798,6 +799,49 @@ Here is another example JWT not only proving possession of a private key but als
 }
 ```
 
+#### `ldp_vp_2.0` Key Proof Type {#ldp_vp-proof-type}
+
+The verifiable presentation as defined by [@VC_DATA_2.0] signed using Data Integrity that are used as Key Proofs MUST contain the following elements:
+
+  * in the presentation itself,
+      * `holder`: OPTIONAL. MUST be equivalent to the DID in the `proof.verificationMethod` property.
```

Comment:
does not need to be a DID - just the verificationMethod identifier

Reply:
That's right. I changed that.